src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-pipe-modal/rgw-multisite-sync-pipe-modal.component.ts

```diff
@@ -110,7 +117,18 @@ export class RgwMultisiteSyncPipeModalComponent implements OnInit {
     }
   }
 
+  getZoneTypeData(zoneDataToFilter: string[], zoneDataForCondition: string[]) {
```

Comment:
nitpick:
suggestion:
```suggestion
  getZoneData(zoneDataToFilter: string[], zoneDataForCondition: string[]) {
```